mapper/orm-coordination-database-polling/src/main/java/org/hibernate/search/mapper/orm/coordination/databasepolling/cfg/HibernateOrmMapperDatabasePollingSettings.java

```diff
@@ -131,6 +131,24 @@ private HibernateOrmMapperDatabasePollingSettings() {
 	public static final String COORDINATION_PROCESSORS_INDEXING_BATCH_SIZE =
 			PREFIX + Radicals.COORDINATION_PROCESSORS_INDEXING_BATCH_SIZE;
 
+	/**
+	 * In the background indexing processor, the timeout for transactions processing outbox events.
+	 * <p>
+	 * Only available when {@link HibernateOrmMapperSettings#COORDINATION_STRATEGY} is
+	 * {@link CoordinationStrategyNames#DATABASE_POLLING}.
```

Comment:
I removed `CoordinationStrategyNames` in a PR that I just merged, so you will need to change this:

```suggestion
	 * {@value #COORDINATION_STRATEGY_NAME}.
```